src/main/scala/com/cognite/spark/datasource/TimeSeriesRelation.scala

```diff
@@ -38,7 +49,34 @@ case class PostTimeSeriesItem(
     assetId: Option[Long],
     isStep: Boolean,
     description: Option[String],
-    securityCategories: Option[Vector[Long]])
+    securityCategories: Option[Vector[Long]],
+    id: Long)
+
+case class UpdateTimeSeriesItem(
+    id: Long,
+    name: Option[Setter[String]],
+    metadata: Option[Setter[Map[String, String]]],
+    unit: Option[Setter[String]],
+    assetId: Option[Setter[Long]],
+    description: Option[Setter[String]],
+    securityCategories: Option[Map[String, Option[Vector[Long]]]],
+    isString: Option[Setter[Boolean]],
+    isStep: Option[Setter[Boolean]]
+)
+object UpdateTimeSeriesItem {
+  def fromPostTimeSeriesItem(postTimeSeriesItem: PostTimeSeriesItem): UpdateTimeSeriesItem =
+    new UpdateTimeSeriesItem(
+      postTimeSeriesItem.id,
+      Setter[String](Some(postTimeSeriesItem.name)),
+      Setter[Map[String, String]](postTimeSeriesItem.metadata),
+      Setter[String](postTimeSeriesItem.unit),
+      Setter[Long](postTimeSeriesItem.assetId),
+      description = Setter[String](postTimeSeriesItem.description),
+      None,
```

Comment:
Why is this (securityCategories) always set to `None`? Can't be changed? Worth a comment at least.